tests/integration/targets/ios_bgp_address_family/tests/cli/_remove_config.yaml

```diff
@@ -18,6 +18,11 @@
 #   ansible.netcommon.cli_config:
 #     config: "{{ lines }}"
 
+- name: Some test data remove later
```

Comment:
This change can be removed, thanks for helping me narrow it down.

Reply:
Sure. Thanks, Sagar.